integration-tests/jbang/src/test/java/io/quarkus/jbang/RunJBangTest.java

```diff
@@ -0,0 +1,155 @@
+package io.quarkus.jbang;
+
+import static io.quarkus.bootstrap.util.PropertyUtils.isWindows;
```

Comment:
@stuartwdouglas any ideas on why the folder still can't be deleted when quarkus have exit'ed?

Reply:
ah just realized it - test is unzipping but not closing the stream. will fix.